src/app/main.cpp

```diff
@@ -0,0 +1,74 @@
+#include "linux_serial_ccsds/linux_serial_ccsds.h"
+#include <cstdio>
+#include <unistd.h>
+
+#include "Thread.h"
+
+static constexpr uint8_t TEXT1[]{ "Hello\n\r" };
+static constexpr uint8_t TEXT2[]{ "Goodbye\n\r" };
+
+static constexpr int SEND_THREAD_PRIORITY1 = 1;
+static constexpr int SEND_THREAD_STACK_SIZE1 = 65536;
+
+static constexpr int SEND_THREAD_PRIORITY2 = 1;
+static constexpr int SEND_THREAD_STACK_SIZE2 = 65536;
+
+linux_serial_ccsds_private_data serial1{};
+linux_serial_ccsds_private_data serial2{};
+
+void* bus_to_driver_private_data[16];
```

Comment:
Ask @rphh about this - i just used 16 as a top of the head number while we was debugging build